packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/pages/search/[[...alias]].jsx

```diff
@@ -0,0 +1,92 @@
+import { NextSeo } from 'next-seo';
+import { isMultiLanguage } from '../../lib/isMultiLanguage.js';
+import {
+	getCurrentLocaleStore,
+	globalDrupalStateStores,
+} from '../../lib/stores';
+import { getDrupalSearchResults } from '@pantheon-systems/drupal-kit';
+import Layout from '../../components/layout';
+import PageHeader from '../../components/page-header';
+
+export default function SearchPage({ hrefLang, footerMenu, errorMessage }) {
+	return (
+		<Layout footerMenu={footerMenu}>
+			<NextSeo
+				title="Decoupled Next Drupal Demo"
+				description="Generated by create next app."
+				languageAlternates={hrefLang || false}
+			/>{' '}
+			<PageHeader title="Search Results" />
+			<div className="mt-12 mx-auto max-w-[50vw]">
+				{errorMessage ? (
+					<p className="text-xl text-center">
+						⚠️Unable to fetch your search results⚠️
+					</p>
+				) : null}
+			</div>
+		</Layout>
+	);
+}
+
+export async function getServerSideProps(context) {
+	const origin = process.env.NEXT_PUBLIC_FRONTEND_URL;
+	const {
+		locales,
+		locale,
+		res,
+		query: { alias },
+	} = context;
+
+	// if there is more than one language in context.locales,
+	// assume multilanguage is enabled.
+	const multiLanguage = isMultiLanguage(locales);
+	const hrefLang = locales.map((locale) => {
+		return {
+			hrefLang: locale,
+			href: origin + '/' + locale,
+		};
+	});
+
+	const store = getCurrentLocaleStore(locale, globalDrupalStateStores);
+
+	try {
+		const footerMenu = await store.getObject({
+			objectName: 'menu_items--main',
+			refresh: true,
+			res: context.res,
+			anon: true,
+		});
+
+		const searchResults = alias
+			? await getDrupalSearchResults(
+					process.env.BACKEND_URL,
+					locale,
+					alias[0],
+					res,
+			  )
+			: undefined;
```

Comment:
**being picky here and this doesn't have to change**: I think I prefer `null` to undefined in general but I'm not sure it really matters!